src/components/tabs/AstroJSXTabs.astro

```diff
@@ -6,12 +6,12 @@ import Tabs from './Tabs';
 	<Fragment slot="tab.1.jsx">JSX</Fragment>
 	<Fragment slot="tab.2.astro">Astro</Fragment>
 
-	<Fragment slot="panel.1.jsx"
-		><slot name="1.jsx" />
+	<Fragment slot="panel.1.jsx">
+		<slot name="1.jsx" />
 		<slot name="jsx" />
 	</Fragment>
-	<Fragment slot="panel.2.astro"
-		><slot name="2.astro" />
+	<Fragment slot="panel.2.astro">
+		<slot name="2.astro" />
```

Comment:
Hmm, that one can cause visual changes, I'm not sure why the formatter changed it automatically, afaik our hugging logic shouldn't change this unless you changed it manually